plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/catalog/rest/RestCatalogTestUtils.java

```diff
@@ -27,15 +27,89 @@
 import org.apache.hadoop.fs.Path;
 import org.apache.iceberg.CatalogProperties;
 import org.apache.iceberg.catalog.Catalog;
+import org.apache.iceberg.catalog.Namespace;
+import org.apache.iceberg.catalog.TableIdentifier;
+import org.apache.iceberg.catalog.ViewCatalog;
+import org.apache.iceberg.inmemory.InMemoryCatalog;
+import org.apache.iceberg.io.FileIO;
 import org.apache.iceberg.jdbc.JdbcCatalog;
+import org.apache.iceberg.jdbc.JdbcClientPool;
+import org.apache.iceberg.view.View;
+import org.apache.iceberg.view.ViewBuilder;
 import org.assertj.core.util.Files;
 
 import java.io.File;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
 
 public final class RestCatalogTestUtils
 {
     private RestCatalogTestUtils() {}
 
+    static class TestingJdbcCatalog
+            extends JdbcCatalog implements ViewCatalog
```

Comment:
If the `JdbcCatalog` at Iceberg supports view. Then we can use it directly instead of implementing `ViewCatalog` here?

There is a PR at Iceberg side to support view for JDBC catalog
https://github.com/apache/iceberg/pull/9487

Reply:
Yes exactly! The only reason JDBC catalog is required in the Trino tests is that catalog for some reason accepts arbitrary objects as configuration. The Trino tests pass in Hadoop config required for the file system configuration to the JDBC catalog.

InMemory would be much better for the purpose of the Trino tests but since it doesn't accept arbitrary objects for config (which is right imo) then we have to use the JDBC catalog.

 If the JDBC catalog makes it in for 1.5 that would be great because then I shouldn't really need any changes here.